keps/sig-node/4009-add-cdi-devices-to-device-plugin-api/kep.yaml

```diff
@@ -0,0 +1,44 @@
+title: Add CDI devices to device plugin API
+kep-number: 4009
+authors:
+  - "@elezar"
+owning-sig: sig-node
+participating-sigs: []
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+creation-date: 2023-05-16
+reviewers:
+  - TBD
+approvers:
+  - TBD
+
+see-also:
+  - "/keps/sig-node/3063-dynamic-resource-allocation"
+  - "/keps/sig-node/3573-device-plugin"
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha|beta|stable
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.28"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.28"
+  beta: "v1.29"
+  stable: "v1.30"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: MyFeature
+    components:
+      - kube-apiserver
+      - kube-controller-manager
+disable-supported: true
+
+# The following PRR answers are required at beta release
+metrics:
+  - my_feature_metric
```

Comment:
If you plan to have a metric corresponding to the feature please add its name here. For alpha, I believe we can leave this empty but it is mandatory for beta graduation.

Reply:
OK. Removing for now. Do you have links to existing metrics for the device plugin?

Reply:
Here are some existing device plugin metrics : https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/metrics/metrics.go#L73-L74.